From c1cf9c6b8dee64ac086e116477e5fec66592dec2 Mon Sep 17 00:00:00 2001
From: Kever Yang <kever.yang@rock-chips.com>
Date: Fri, 28 Dec 2018 17:27:21 +0800
Subject: [PATCH] rockchip: sip: add static inline for
 sip_smc_lastlog_request()

Add static inline for sip_smc_lastlog_request() when
CONFIG_ROCKCHIP_SIP is not enabled.

Change-Id: I8fe46c36b115b4aa8a71a4101a7829e64aa08103
Signed-off-by: Kever Yang <kever.yang@rock-chips.com>
---
 include/linux/rockchip/rockchip_sip.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/rockchip/rockchip_sip.h b/include/linux/rockchip/rockchip_sip.h
index 3258eeda6e50..2f25be819f5d 100644
--- a/include/linux/rockchip/rockchip_sip.h
+++ b/include/linux/rockchip/rockchip_sip.h
@@ -180,7 +180,7 @@ sip_smc_vpu_reset(u32 arg0, u32 arg1, u32 arg2)
 	return tmp;
 }
 
-struct arm_smccc_res sip_smc_lastlog_request(void)
+static inline struct arm_smccc_res sip_smc_lastlog_request(void)
 {
 	struct arm_smccc_res tmp = {0};
 	return tmp;
-- 
2.35.3

